gaps/gap-25_golem_certificates/gap-25_golem_certificates.md

```diff
@@ -30,46 +31,68 @@ When designing certificates structure that will allow to meet requirements menti
 - KYC
 - Inbound network traffic
 
-This means that certificates will be used to sign different kinds of permissions. In case of transferring certification to other companies or signing legal Agreements, we need to ensure that we keep control which permissions they will be able to use.
+This means that certificates will be used to sign different kinds of permissions. In case of transferring certification to other companies or signing legal Agreements, we need to ensure that we control which permissions they will be able to use.
 
-In the future of Golem Network we expect that new, independent features will be implemented by community and they may need to use certification. This document aims to standardize this approach.
+In the future of Golem Network we expect that new, independent features will be implemented by the community and they may need to use certification. This document aims to standardise this approach.
```

Comment:
out of curiosity - do we have guidelines that would force us to standardize on British variants? if not, I wouldn't introduce changes that serve only to change the American English spellings to British ones...